common/customproperties.go

```diff
@@ -19,7 +19,7 @@ type CustomProperties struct {
 	x *custom_properties.Properties
 }
 
-// CustomProperty contains document specific property
+// Using of this type is deprecated.
```

Comment:
Can you make this comment fulfill godoc expectations?  And add a remark that it will be deprecated in a future major version.  Similar to what we have done before.

So we want to use `custom_properties.CT_Property` instead of this CustomProperty?
Might have been better to define a struct to contain this type and make accessible via .X() function as in some other models we have.

Reply:
fixed